reciprocalspaceship/io/crystfel.py

```diff
@@ -0,0 +1,190 @@
+import pandas as pd
+from reciprocalspaceship import DataSet
+import numpy as np
+
+
+def _parse_stream(filename: str) -> dict:
```

Comment:
Nice parser -- clear to read. One revision -- please change the docstring to match that of `read_crystfel()`.

Reply:
I didn't fully understand your remark regarding the docstring -- please check whether it's done as you intended.